spyder/plugins/editor/widgets/gotoline.py

```diff
@@ -75,10 +77,14 @@ def __init__(self, editor):
     def text_has_changed(self, text):
         """Line edit's text has changed."""
         text = str(text)
-        if text:
+
+        # QIntValidator does not handle '+' sign
+        # See Spyder PR #20070
```

Comment:
Lets change this to:
```suggestion
        # See spyder-ide/spyder#12693
```
To follow the link format as above and to point to the initial issue: https://github.com/spyder-ide/spyder/issues/12693